views/partials/forms/option-group.html

```diff
@@ -0,0 +1,28 @@
+<div id="{{key}}-group" class="form-group{{#className}} {{className}} {{/className}}{{#formGroupClassName}} {{formGroupClassName}}{{/formGroupClassName}}{{#error}} validation-error{{/error}}">
+    <fieldset role="{{role}}"{{#ariaRequired}} aria-required="true"{{/ariaRequired}}>
+        <legend>
+            <span{{#legendClassName}} class="{{legendClassName}}"{{/legendClassName}}>{{legend}}</span>
+            {{#hint}}<span id="{{key}}-hint" class="form-hint">{{hint}}</span>{{/hint}}
+            {{#error}}<span id="{{key}}-error" class="error-message">{{error.message}}</span>{{/error}}
+        </legend>
+        {{#options}}
+        <div class="multiple-choice">
```

Comment:
This template overrides https://github.com/UKHomeOfficeForms/hof-template-mixins/blob/master/partials/forms/option-group.html

Adds the the multiple-choice class from the new gov.uk elements v3
and change of the layout of the template because of the way this class works